stock_picking_report_valued/readme/USAGE.rst

```diff
@@ -0,0 +1,12 @@
+To get the stock picking valued report:
+
+#. Create a Sale Order with stockable products a *Valued picking* able
+   customer.
+#. Confirm the Sale Order.
+#. Click on *Deliveries* stat button.
+#. Go to *Print > Delivery Slip*.
+
```

Comment:
I think you should remove the last lines and just let the usage description. See https://github.com/OCA/maintainer-tools/blob/master/template/module/readme/USAGE.rst